Harvest.Jobs.Invoice/Program.cs

```diff
@@ -59,6 +60,7 @@ private static ServiceProvider ConfigureServices()
             services.AddScoped<ISlothService, SlothService>();
             services.AddScoped<IFinancialService, FinancialService>();
             //services.Configure<SparkpostSettings>(Configuration.GetSection("Sparkpost"));
+            services.AddScoped(provder => StandardJsonOptions.GetOptions());

```

Comment:
To reignite this debate for fun, why scoped?  it's this really just a singleton?  The options never changes or needs to be re-allocated

Reply:
Yeah, I was just doing it because the services it was going into are scoped. I'll change it to singleton...